### PR TITLE
Support for PySM and libsharp in the satellite simulation pipeline

### PR DESCRIPTION
**Pull request by zonca**

### CODE REVIEW

File: pipelines/toast_satellite_sim.py [hunk 1]
```diff
@@ -553,6 +663,9 @@ def main():
             nse = tt.OpSimNoise(out="noise", realization=mc)
             nse.exec(data)
 
+            # add sky signal
+            add_sky_signal(args, comm, data, totalname="noise", signalname=signalname)
```
Comment (zonca) [inline pipelines/toast_satellite_sim.py]: do you prefer I rename the key to "signal" now?

**Comment by zonca** [comment]
@tskisner @keskitalo this is ready for a first review.

Should I also implement the capability of rescanning an existing map?

File: pipelines/toast_satellite_sim.py (hunk 1 above)
Comment (tskisner) [reply pipelines/toast_satellite_sim.py]: Sure, that would make sense.

**Comment by tskisner** [comment]
I've not tested the modifications, but they look fine to me.

File: pipelines/toast_satellite_sim.py (hunk 1 above)
Comment (zonca) [reply pipelines/toast_satellite_sim.py]: ok, I will rename it to `tot_signal` because `signal` is already the sky signal

**Comment by zonca** [comment]
ok @tskisner, I renamed the object in the cache and tested on `tiny_satellite_shell.sh`

**Pull request merged**